src/todosRepository.js

```diff
@@ -0,0 +1,16 @@
+import * as ko from 'knockout';
+import { Todo } from './todo.model.js';
+
+export const todosRepository = {
+    load() {
+        console.log(window.localStorage);
+        let list = JSON.parse(localStorage.getItem('todoList'));
```

Comment:
`JSON.parse(localStorage.getItem('todoList')) || []`
will let you avoid ternary operator below. It will assign empty array to list if localStorage contains null